CHANGELOG.md

```diff
@@ -360,11 +359,6 @@ private fun LockedScreen() {
 }
 ```
 
-> **Note**
-> Android paywalls is currently behind an experimental flag: `ExperimentalPreviewRevenueCatUIPurchasesAPI`.
-> It is safe to release app updates with it. We guarantee that paywalls will continue to work and any changes will always be backwards compatible.
-> They are stable, but migration steps may be required in the future. We'll do our best to minimize any changes you have to make.
```

Comment:
Hmm should we remove this from the CHANGELOG? I think I would keep it and just add the change in the CHANGELOG for the new version.

Reply:
ugh my bad, this is me being stupid and not checking where I was removing this